packages/eb-deep-merge/package.json

```
@@ -1,4 +1,7 @@
 {
+  "author": "Empathy Systems Corporation S.L.",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/empathyco/x/tree/main/packages/x-deep-merge#readme",
   "name": "@empathyco/x-deep-merge",
   "version": "1.2.1",
   "description": "Clones an array of objects into another object.",
```

I'd change this description to `Utility for deep cloning objects.`

```suggestion
  "description": "Utility for deep cloning objects.",
```